csharp/Adapter/Microsoft.Spark.CSharp/Network/ISocketWrapper.cs

```diff
@@ -38,6 +38,18 @@ internal interface ISocketWrapper : IDisposable
         Stream GetStream();
```

Comment:
Tao Wang introduced the input and output stream at the CSharpWorker only. I do not do lots of changes on drive side. So still keep the function here.